docs/Governance.md

```diff
@@ -115,6 +115,17 @@ Responsibilities:
 * Manage github settings such as branch protection rules and repository secrets.
 * Resolve any disputes among maintainers.
 
+## Prioritizing Issues
+
+In the weekly triage meetings, new issues are assigned to contributors and are set to a monthly milestone by which the issue is expected to be completed. It may not be possible for the assignees to address all the issues assigned to them by their target milestone. As a result issues may pile up in a given milestone. In such cases project admins can request contributors to use *special labels* to prioritize issues as follows:
```

Comment:
I think it can't be Contributors who use special labels (as lines 15-20 explain anyone can submit a PR but they don't have permission to manage labels), but instead it needs to be someone who is a Triage Driver or Maintainer who should use such labels.